src/lib/bootloader/systeminfo.rb

```diff
@@ -108,6 +118,31 @@ def s390_secure_boot_active?
       rescue StandardError
         false
       end
+
+      # The partition where zipl is installed.
+      #
+      # @return [Y2Storage::Partition, NilClass] zipl partition
+      def zipl_device
+        staging = Y2Storage::StorageManager.instance.staging
+        mountpoint = Y2Storage::MountPoint.find_by_path(staging, "/boot/zipl").first
+        mountpoint.filesystem.blk_devices.first
+      rescue StandardError
+        nil
+      end
+
+      # Check if device is a SCSI device.
+      #
+      # param device [Y2Storage::Partition, NilClass] partition device (or nil)
+      #
+      # @return [Boolean] true if device is a SCSI device
+      def scsi?(device)
+        # in lack of a better idea: check if device name starts with 'sd'
+        # alternatively: device.udev_ids.any?(/^scsi-/)
+        # or: device.udev_paths.any?(/-zfcp-/)
+        device.name.start_with?("/dev/sd")
```

Comment:
hmm, name check is a bit strange. I expect that such kind of info storage-ng contain. @ancorgs @aschnell is there such method for device? Or do you use hwinfo for it?

Reply:
The info does not exist. I've asked IBM if this would do and they agreed.

Reply:
We have `Disk#transport` that can be one of these: `UNKNOWN, SBP, ATA, FC, ISCSI, SAS, SATA, SPI, USB, FCOE`

And we also have `BlkDevice#bus` that returns the bus reported by hwinfo and, more generally, `BlkDevice#hwinfo` that contains all the information from hwinfo.

I kind of expect that any of those can be used.

Reply:
Interesting. I Looked at the XML and found no usable data there.

Reply:
Some more thoughts:

- `transport` is the scsi transport layer, so we might go for `FC` and hope that's our use-case here; IMHO testing for scsi in general looks more correct (and also easier testable)
- `bus` is not in libstorage-ng and so not logged in the device graph XML; that again makes it hard to debug as the actual value shows up nowhere in any logs